Dockerfile

```diff
@@ -16,28 +16,32 @@
 # purpose image. It contains the toolchains for all supported architectures, and
 # all build dependencies.
 FROM registry.fedoraproject.org/fedora-minimal:35 AS build
+ENV GCC_VERSION=11.2.1-1.fc35
+ENV NASM_VERSION=2.15.05-1.fc35
+ENV PYTHON_VERSION=3.10
```

Comment:
Just a heads up, there is a known issue with python 3.9+ that may break ARM64 build due to an issue pulling in the ARM compiler zip file. The fix has been merged into 3.10, but it doesn't seem they have released a tag including it yet.  

https://github.com/python/cpython/issues/91387 

I'm not sure if this is a blocker at the moment, but in case you hit this.

Reply:
Oh, good to know! So far it works in the CI at least.
I disabled pulling compilers at runtime in my patch set that enables containers in the CI.
But someone using the image locally might run into this.